tensorflow/c/experimental/filesystem/modular_filesystem.cc

```diff
@@ -285,9 +286,8 @@ Status ModularFileSystem::CreateDir(
   return StatusFromTF_Status(plugin_status.get());
 }
 
-Status ModularFileSystem::Stat(
-    const std::string& fname,
-    FileStatistics* stat /*, TransactionToken* token */) {
+Status ModularFileSystem::Stat(const std::string& fname, FileStatistics* stat,
+                               TransactionToken* token) {
```

Comment:
For these methods `token` is an input argument, right? It should come before `stat` which is output